perception/autoware_lidar_bevfusion/src/lidar_bevfusion_node.cpp

```
@@ -0,0 +1,341 @@
+
+  // Camera branch parameters
+  const auto d_bound =
+    to_float_vector(this->declare_parameter<std::vector<double>>("d_bound", descriptor));
+  const auto x_bound =
+    to_float_vector(this->declare_parameter<std::vector<double>>("x_bound", descriptor));
+  const auto y_bound =
+    to_float_vector(this->declare_parameter<std::vector<double>>("y_bound", descriptor));
+  const auto z_bound =
+    to_float_vector(this->declare_parameter<std::vector<double>>("z_bound", descriptor));
+  const auto num_cameras = this->declare_parameter<std::int64_t>("num_cameras", descriptor);
+  const auto raw_image_height =
+    this->declare_parameter<std::int64_t>("raw_image_height", descriptor);
+  const auto raw_image_width = this->declare_parameter<std::int64_t>("raw_image_width", descriptor);
+  const auto img_aug_scale_x = this->declare_parameter<float>("img_aug_scale_x", descriptor);
+  const auto img_aug_scale_y = this->declare_parameter<float>("img_aug_scale_y", descriptor);
+  const auto roi_height = this->declare_parameter<std::int64_t>("roi_height", descriptor);
+  const auto roi_width = this->declare_parameter<std::int64_t>("roi_width", descriptor);
+  const auto features_height = this->declare_parameter<std::int64_t>("features_height", descriptor);
+  const auto features_width = this->declare_parameter<int>("features_width", descriptor);
+  const auto num_depth_features = this->declare_parameter<int>("num_depth_features", descriptor);
+
+  // Head parameters
+  const auto num_proposals = this->declare_parameter<std::int64_t>("num_proposals", descriptor);
+  class_names_ = this->declare_parameter<std::vector<std::string>>("class_names", descriptor);
+
+  if (point_cloud_range.size() != 6) {
+    RCLCPP_WARN_STREAM(
+      rclcpp::get_logger("lidar_bevfusion"),
+      "The size of point_cloud_range != 6: use the default parameters.");
+  }
+  if (voxel_size.size() != 3) {
+    RCLCPP_WARN_STREAM(
```

For `voxel_size` and `point_cloud_range` it should be `RCLCPP_ERROR_STREAM` and maybe throw an exception.